docker-compose.yml

```diff
@@ -1,13 +1,14 @@
-version: '3'
+version: '3.4'
 services:
-  tests: &tests-config
+  release:
```

Comment:
tohle je ten release - důležitá věc je, že NEMÁ MOUNT. Takže skutečně spouštíš to, co je v tom image, ne to co jsi namountoval.